### PR TITLE
Update localizations

### PR DESCRIPTION
**Pull request by Eitot**
- Update several Info.plist keys
- Add descriptions to Info.plist localizable keys
- Replace a localized key

### CODE REVIEW

File: Vienna/Resources/Base.lproj/InfoPlist.strings [hunk 1]
```diff
@@ -1 +1,9 @@
-/* Do not remove */
+/* A message that tells the user why Vienna is requesting the ability to send Apple events. */
+"NSAppleEventsUsageDescription" = "";
+
+/* A label for plug-in bundles. This is shown in Finder's Get Info window, for example. */
+"Vienna plug-in" = "Vienna plug-in";
```
Comment (TAKeanice) [inline Vienna/Resources/Base.lproj/InfoPlist.strings:5]: Shouldn't those lines have the same keys as the info.plist?

File: Vienna/Resources/Base.lproj/InfoPlist.strings [hunk 2]
```diff
@@ -1 +1,9 @@
...
+
+/* A label for style bundles. This is shown in Finder's Get Info window, for example. */
+"Vienna style" = "Vienna style";
```
Comment (TAKeanice) [inline Vienna/Resources/Base.lproj/InfoPlist.strings:8]: Same here, shouldn't the key be "Style"?

**Review by TAKeanice — CHANGES_REQUESTED** [review]

File: Vienna/Resources/Base.lproj/InfoPlist.strings (hunk 1 above)
Comment (Eitot) [reply Vienna/Resources/Base.lproj/InfoPlist.strings:5]: Nope. The "Plug-in" and "Style" values are non-localisable and only appear in the auto-generated document icons. These keys here are the values for the document description that appear in Finder.

**Review by TAKeanice — DISMISSED** [review]

**Comment by Eitot** [comment]
I still have to import the localisations before this is merged.

**Comment by TAKeanice** [comment]
What about localizations for the WebKit browser? Are they included? I am not sure whether I have introduced any new keys

**Comment by Eitot** [comment]
It is based on `master`, I think there are just three new strings.

**Pull request merged**